numpy/core/src/common/npy_cpu_features.c

```diff
@@ -0,0 +1,727 @@
+#line 1 "numpy/core/src/common/npy_cpu_features.c.src"
+
+/*
+ *****************************************************************************
+ **       This file was autogenerated from a template  DO NOT EDIT!!!!      **
+ **       Changes should be made to the original source (.src) file         **
+ *****************************************************************************
+ */
```

Comment:
This should not be committed, right?

I'd suggest you just spawn a separate PR to ignore this, so we can get it out of the way.

Reply:
Yes, that was a mistake.